likecoin/admin/views/matters.php

```diff
@@ -22,6 +22,40 @@
 
 // phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
 
+/**
+ * Generate matters introduction
+ */
+function likecoin_add_matters_introduction() {
+	?>
+	<div>
+	<p>
+		<h2><a href="https://matters.news">
+		<img height="32" weight="32" src="https://matters.news/static/icon-144x144.png"></img>
+		</a>What is matters.news?</h2>
```

Comment:
Why not wrap the entire `<h2>` content with `<a>`, because of the underline?
btw, `<img>` dont have close tag

Reply:
Because I think ppl might click on the icon, but the title is not suppose to be clickable (strange reason I guess...)